eth/tracers/api_test.go

```diff
@@ -547,6 +550,111 @@ func TestTraceTransaction(t *testing.T) {
 	}
 }
 
+func TestTraceBlockFromFile(t *testing.T) {
+	t.Parallel()
+
+	// Set up test accounts and a simple genesis block
+	accounts := newAccounts(2)
+	genesis := &core.Genesis{
+		Config: params.TestChainConfig,
+		Alloc: types.GenesisAlloc{
+			accounts[0].addr: {Balance: big.NewInt(params.Ether)},
+		},
+	}
+	genBlock := 3
+	signer := types.HomesteadSigner{}
+	backend := newTestBackend(nil, genBlock, genesis, func(i int, b *core.BlockGen) {
```

Comment:
This is a good start but the blocks will have empty traces. It will be great if some code is executed too as part of a tx in there.